### PR TITLE
Display size in Human readable format.

### LINKED ISSUE #73: Display size in human readable format in the UI
``34443782 bytes`` is quite hard for me to parse.
After some calculation I understood it was ``32,85 MB`` I think that's the value I would like to have in the UI directly. We could however display the bytes value on mouseover (title?acronym?)

### PR DESCRIPTION
**Pull request by Natim**
Fixes #73

### CODE REVIEW

File: ui/Model.elm [hunk 1]
```diff
@@ -33,6 +33,7 @@ getBuildRecordList =
     client
         |> Kinto.getList recordResource
         |> Kinto.sortBy [ "-build.date" ]
+        |> Kinto.limit 40
```
Comment (Natim) [inline ui/Model.elm]: Get rid of this.

File: ui/View.elm [hunk 2]
```diff
@@ -283,7 +289,7 @@ viewDownloadDetails download =
                     [ td [] [ a [ href download.url ] [ text filename ] ]
                     , td [] [ text <| download.mimetype ]
                       -- TODO display the size in an humanly readable format
-                    , td [] [ text <| toString download.size ]
+                    , td [] [ text <| format download.size ]
```
Comment (magopian) [inline ui/View.elm]: you can also remove the comment now ;)

**Comment by Natim** [comment]
Refs https://github.com/basti1302/elm-human-readable-filesize/pull/1

File: ui/View.elm [hunk 3]
```diff
@@ -282,8 +288,7 @@ viewDownloadDetails download =
                 [ tr []
                     [ td [] [ a [ href download.url ] [ text filename ] ]
                     , td [] [ text <| download.mimetype ]
-                      -- TODO display the size in an humanly readable format
-                    , td [] [ text <| toString download.size ]
+                    , td [] [ text <| format download.size ]
```
Comment (glasserc) [inline ui/View.elm]: I think `format` is too general a name for this. How about `formatFilesize`?

Comment (Natim) [inline ui/View.elm]: Yes sure.

**Pull request merged**